src/agoradatatools/etl/transform/immunohisto_transform.py

```diff
@@ -24,18 +92,28 @@ def immunohisto_transform(
     Args:
         datasets (Dict[str, pd.DataFrame]): Dictionary of dataset names mapped to their DataFrame.
         dataset_name (str): The name of the dataset to transform.
-        group_columns (List[str], optional): List of columns to group by. Defaults to ['model', 'type', 'age_death', 'tissue', 'units'].
-        extra_columns (List[str], optional): List of columns to include in the group. Defaults to ['genotype', 'measurement', 'sex'].
-        extra_column_name (str, optional): Name of the column containing the extra columns. Defaults to 'points'.
+        group_columns (List[str], optional): List of columns to group by. Defaults to ['model', 'evidence_type', 'tissue', 'age', 'units'].
+        extra_columns (List[str], optional): List of columns to include in the group. Defaults to ['genotype', 'sex', 'individual_id', 'value'].
+        extra_column_name (str, optional): Name of the column containing the extra columns. Defaults to 'data'.
 
     Returns:
         pd.DataFrame: A DataFrame grouped by the group_columns.
```

Comment:
Could you add a little more information to both the Returns comment and the initial description about the structure of the output data frame? I think something like: "A DataFrame containing all group_columns, plus an extra column named as specified in extra_column_name. This extra column contains all information from extra_columns, collapsed into a single dictionary."